File: cp_nim/src/bin/abc444-a-repdigit.nim
# Created by Ayush Biswas at 2026/07/18 18:53
# https://atcoder.jp/contests/adt_easy_20260508_1/tasks/abc444_a
import lib/cpio

# @code begin

import sequtils

var r = newReader()

let s = r[string]

if s.deduplicate(false).len == 1:
  echo "Yes"
else:
  echo "No"

# @code end

File: cp_rust/src/bin/abc444-a-repdigit.rs
// Created by Ayush Biswas at 2026/07/07 20:59
// https://atcoder.jp/contests/adt_easy_20260508_1/tasks/abc444_a
use cp_lib::*;

// @code begin
use cpio::*;
use itertools::Itertools;

sol! {
    fn solution(
        n: [char]
    ) -> Bool {
        (n.into_iter().group_by(|&x| x).count() == 1).into()
    }
}
// @code end
